db/schema.rb

```diff
@@ -0,0 +1,59 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
Your database schema is not following the suggested one. Please make the corresponding migrations.

Check the images below for a better understanding:
![Capture5](https://user-images.githubusercontent.com/75709904/149641138-0dd7db61-50c4-4381-b6ea-8d7e2c089092.PNG)
![Capture6](https://user-images.githubusercontent.com/75709904/149641140-8fb93b8f-7b01-489b-94a4-092239a54398.PNG)